### ISSUE TITLE
Rename project OP2TilesetConverter

### ISSUE THREAD

**Issue by Brett208** [OutpostUniverse/OP2BitmapConverter#13]
I don't think we plan to support converting non-tileset bitmaps. I probably chose a bad name to start with.

**Comment by DanRStevens** [comment]
Right. Well, I suppose we're looking to convert to/from the custom bitmap format used by OP2, which happens to be used for tilesets. Maybe OP2CustomBitmapConverted? Though really I'm fine with the proposed name change. People probably will be thinking about tilesets when they use it.

**Comment by Brett208** [comment]
Yeah, I was thinking that since it doesn't actually convert any of the sprites besides the tilesets, the current name would be confusing as someone might think they could use this project to convert other game art assets. To my knowledge, this project would only be useful for the tilesets since the other art assets are stored in a separate manner.
